addons/binding/org.openhab.binding.elerotransmitterstick/.classpath

```diff
@@ -3,5 +3,7 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="lib" path="lib/commons-codec-1.8.jar"/>
+	<classpathentry kind="lib" path="lib/commons-lang3-3.4.jar"/>
```

Comment:
Unless you really cannot live without version 3 please stick to the commons library we provide in the the targetplattform, see our discussion here: https://github.com/eclipse/smarthome/issues/3522

Also bindings should ship as little external libraries as possible.

Reply:
I don't care about the version. It was just that a user reported that the binding was not working for him due to missing commons-codec. I then checked https://www.eclipse.org/smarthome/documentation/development/bindings/dependencies.html, which I understood as no apache libs are included, and I have to include them if I need them. Where can I see which libraries are part of ESH/openHAB so that I do not have to include them?

Reply:
It could indeed be that smarthome itself has less supported lies but we have some, see https://docs.openhab.org/developers/development/guidelines.html#c-language-levels-and-libraries